server/server.js

```diff
@@ -1,7 +1,6 @@
 const express = require('express');
-const db = require('./db/index');
-
-// const path = require('path');
+const db = require('../db/index');
```

Comment:
I believe you need to put index.js rather than just index, or leave it as /db to automatically look for the index file. Please let me know if this isn't the case.

Reply:
This is currently working for me.